src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue

```diff
@@ -249,9 +250,15 @@ function getPickedTokens(tokens: PoolToken[]) {
           <BalAssetSet :logoURIs="orderedTokenURIs(gauge)" :width="100" />
         </div>
       </template>
-      <template #poolCompositionCell="{ pool, address, addedTimestamp }">
+      <template
+        #poolCompositionCell="{ pool, address, addedTimestamp, network }"
+      >
         <div v-if="!isLoading" class="flex items-center py-4 px-6">
+          <div v-if="POOLS_MAP[network]?.Metadata[pool.id]" class="text-left">
```

Comment:
I suggest extracting this to a semantic method in `pools.ts` (something like `getMetadata(poolId)`)

Reply:
Done.